lang/cs/Org.Apache.REEF.Client/API/JobParametersBuilder.cs

```diff
@@ -89,6 +92,19 @@ public JobParametersBuilder SetDriverMemory(int driverMemoryInMb)
         }
 
         /// <summary>
+        /// Adds job submission environment variable.
+        /// If the variable is already in the map, override it. 
+        /// </summary>
+        /// <param name="key"></param>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        public JobParametersBuilder AddJobSubmissionEnvVariable(string key, string value)
+        {
+            _jobSubmissionMap[key] = value;
```

Comment:
Maybe we should add an optional out bool variable to indicate whether or not an overwrite happened?

Reply:
The usage is like build pattern. The user won't have chance to check the out parameter. Make the parameter like <key, value> is clean and simple. It is client's responsibility to set it. If they want to override, just override.